enterprise/static/enterprise/admin/manage_learners.js

```diff
@@ -18,5 +55,20 @@ $(
                 })
             }
         });
+        $("#id_course").on('input', function(evt){
```

Comment:
@asadiqbal08 @haikuginger technically it should be "space before opening curly-brace". :)